src/components/app_catalog/AppList/AppList.js

```diff
@@ -19,7 +19,7 @@ const AppList = ({ catalog, ...props }) => {
       }}
     >
       <DocumentTitle title={'Apps | Giant Swarm '}>
-        <>
+        <React.Fragment>
```

Comment:
I guess things got a bit confused here, sorry for doing large changes while you're also doing the filename change.

Reply:
I'm going to change this one to the short form in here

Reply:
But I guess the Fragment isn't required at all.. Will remove it